docs/sdks/client-sdks/javascript.md

```diff
@@ -70,16 +70,35 @@ If you are using the SDK for experiment assignments, make sure to pass in an ass
+| **`throwOnFailedInitialization`** (boolean)       | Throw an error (reject the promise) if unable to fetch initial configurations during initialization.                                                                                                                                                                                                                                   | `true`          |
+| **`numPollRequestRetries`** (number)              | If polling for updated configurations after initialization, the number of additional times a request will be attempted before giving up. Subsequent attempts are done using an exponential backoff.                                                                                                                                    | `7`             |
+| **`skipInitialRequest`** (boolean)                | Skip the initial request for a new configuration during initialization (if polling is enabled, this will still take place later)                                                                                                                                                                                                       | `false`         |
+| **`persistentStore`** (IAsyncStore)               | An asynchronous, persistent storage for caching fetched flag configurations for use in subsequent sessions                                                                                                                                                                                                                             | _Eppo provided_ |
+| **`maxCacheAgeSeconds`** (number)                 | Maximum age, in seconds, previously cached configurations are considered valid and fetching an update configurations will be skipped                                                                                                                                                                                                   | `0`             |
+| **`useExpiredCache`** (boolean)                   | Consider initialization successfully complete without fetching updates, even if the configuration loaded from the cache is expired                                                                                                                                                                                                     | `false`         |
+| **`updateOnFetch`** (always\|expired\|empty)      | Sets how the configuration is updated after a successful fetch:<br/>• always - immediately start using the new configuration<br/>• expired - immediately start using the new configuration only if the current one has expired<br/>• empty - only use the new configuration if the current one is both expired and uninitialized/empty | `'always'`      |
```

Comment:
Neat, I didn't think we could fit bulleted lists in these tables!

Reply:
ha yeah a bit of a workaround but it works